cmd/update_container.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"bufio"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/YaleSpinup/spinup-cli/pkg/spinup"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+)
+
+var redeployContainerCmd bool
+
+func init() {
+	updateCmd.AddCommand(updateContainerCmd)
+	updateContainerCmd.PersistentFlags().BoolVarP(&redeployContainerCmd, "redeploy", "r", false, "Redeploy with the current configuraiton.")
+}
+
+var updateContainerCmd = &cobra.Command{
+	Use:   "container",
+	Short: "Update a container resource",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return errors.New("exactly 1 container service id is required")
+		}
+
+		resource := &spinup.Resource{}
+		if err := SpinupClient.GetResource(map[string]string{"id": args[0]}, resource); err != nil {
```

Comment:
I was wondering how that CSRF token was getting set in PutResource, then I noticed you call GetResource here 😸 I guess you need to confirm that the resource is created before doing an update anyway.
What if we ever need to do a PUT (or POST would be the same problem) without a GET first? I don't understand how that token works exactly 😃

Reply:
We would have to do the get first.  The XSRF protection stuff goes away for API endpoints though, so if we start using those instead of web resources, or modify the resources using API authentication not not require it, it won't be a problem anymore.  👍